src/logging.cc

```diff
@@ -26,27 +26,29 @@
 
 #include "triton/common/logging.h"
 
-#ifdef _WIN32
-// suppress the min and max definitions in Windef.h.
-#define NOMINMAX
-#include <Windows.h>
-#else
-#include <sys/time.h>
-#include <sys/types.h>
-#include <time.h>
-#include <unistd.h>
-#endif
 #include <algorithm>
 #include <iomanip>
 #include <iostream>
 
+#include "triton/common/error.h"
+
+// Defined but not used
+#define TRITONJSON_STATUSTYPE uint8_t
+#define TRITONJSON_STATUSRETURN(M)
+#define TRITONJSON_STATUSSUCCESS 0
+
+#include "triton/common/triton_json.h"
+
 namespace triton { namespace common {
 
 Logger gLogger_;
 
 Logger::Logger()
     : enables_{true, true, true}, vlevel_(0), format_(Format::kDEFAULT)
 {
+  const char* value = std::getenv(Logger::ESCAPE_ENVIRONMENT_VARIABLE);
+  escape_log_messages_ =
+      (value && std::strcmp(value, "FALSE") == 0) ? false : true;
```

Comment:
Can we start with 0/1 instead of having to worry about case sensitivity of FALSE/false?

- [TRITON_PARALLEL_INSTANCE_LOADING=1](https://github.com/triton-inference-server/core/blob/0089bb72b4ab616e69acdc49e302af92c921726e/src/backend_model.cc#L559-L570)
- NVIDIA_TF32_OVERRIDE=0
- NCCL_P2P_DIRECT_DISABLE=1